b-lab-org/ember-cli-impact-core
# Remove Abstract B Impact Report [#154842655]

##### Story
* https://www.pivotaltracker.com/story/show/154842655

##### QA
This should be QA'd within the context of the `management` or `impact` app
* `./impact.sh up`
* Register  w/ 0 employees
* Choose Full Impact user
* Earn points in IBM
* Earn some N/A points
* Navigate to B Impact Report
* See IBM highlighted in blue
* Workers section shows `N/A` for score and cannot be expanded
* See row for `N/A` points in impact area where they were earned
* Generally, should match comp here: https://www.pivotaltracker.com/story/show/154842655/comments/185490992
